src/components/EntityInfo.vue

```diff
@@ -265,15 +271,7 @@ export default class EntityInfo extends Mixins(AllowableActionsMixin, CommonMixi
 
   /** The NR description. */
   private get nrDescription (): string {
-    return this.entityStatusToDescription(this.entityStatus, this.getEntityType)
```

Comment:
`entityStatusToDescription` was only used here so I deleted it and moved its code here instead.